README.md

```diff
@@ -34,12 +34,8 @@ you need to follow the steps below:
 -   Build it: `npm install`
 -   Commit the package.json and package-lock.json files, push to a branch, make a PR, have it reviewed and merged to main.
 -   Pull and checkout main on your last commit.
--   [Tag your last commit](https://semver.org/) : `git tag <tag>` (example: `v0.6.0`)
--   Push tag : `git push origin <tag>`
--   Checkout the tag in a fresh repo copy : `cd $(mktemp -d) && git clone https://github.com/gridsuite/commons-ui.git` then `cd commons-ui && git checkout <tag>`
--   [Test your package](https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#testing-your-package): `npm install`
--   [Login on the command line to the npm registry](https://docs.npmjs.com/logging-in-to-an-npm-enterprise-registry-from-the-command-line): `npm login`
--   [Publish the package](https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages): `npm publish`
+-   Make a release on GitHub by creating a new tag on the last commit 
```

Comment:
Maybe we can link to this page to create a release and trigger this action: https://github.com/gridsuite/commons-ui/releases/new ?

Reply:
We still need to tag to release, no?
```
-   [Tag your last commit](https://semver.org/) : `git tag <tag>` (example: `v0.6.0`)
-   Push tag : `git push origin <tag>`
```